dvc/utils/__init__.py

```diff
@@ -382,3 +382,8 @@ def relpath(path, start=os.curdir):
     ):
         return path
     return os.path.relpath(path, start)
+
+
+def chunk(iterable, chunk_size=100):
```

Comment:
Isn't `to_chunks` the same thing?

Reply:
Also, where did 100 come from? Did you estimate it for sql or something?

Reply:
not exactly, in to_chunks we define number of chunks, so chunk length varies depending on length of argument, in this case I am focusing on particular chunk length. As to `chunk_size`  the only criterion taken by me into account was maximum variables number, which, according to [docs](https://www.sqlite.org/limits.html) is 999. Do you think I should perform some performance tests before deciding to default?

Reply:
For sure it should not be default in method. Its wrong to decide generic function default argument basing on particular use case scenario.

Reply:
> not exactly, in to_chunks we define number of chunks, so chunk length varies depending on length of argument, in this case I am focusing on particular chunk length

Got it. :+1: 

>  As to chunk_size the only criterion taken by me into account was maximum variables number, which, according to docs is 999.

Left a comment about it down below.

> Do you think I should perform some performance tests before deciding to default?

Not really, the more - the better in this case, isn't it the point of this whole PR ? :slightly_smiling_face: Let's try to squeeze as much as we can.